src/test/java/com/ibanity/apis/client/http/service/impl/IbanityHttpSignatureServiceImplTest.java

```diff
@@ -126,8 +132,8 @@ private String getRequestPayload() {
         return "{\"msg\":\"hello\"}";
     }
 
-    private Map<String, String> getSignatureHeaders() throws MalformedURLException {
-        URL url = new URL("https://myproxy.com/xs2a/customer-access-tokens?test=1&test=2");
+    private Map<String, String> getSignatureHeaders(String host) throws MalformedURLException {
```

Comment:
host should be renamed basePath or like. It's not a host that we give here, as it includes the scheme

Reply:
Fixed